docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-promotions-and-discounts-feature-walkthrough.md

```diff
@@ -13,8 +12,25 @@ To learn more about the feature and to find out how end users use it, see [Marke
 
 {% endinfo_block %}
 
+## Module dependency graph
+
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/75358e26-725d-4f7d-8686-c72be236b88e.png?utm_medium=live&utm_source=custom)
```

Comment:
It's fine to get rid of parent feature(Promotions & Discounts) modules here, keep only those that belong exclusively to `Marketplace Promotions & Discounts`.
Some of the modules can be kept if they are required for context understanding.

Reply:
got it, removed.